server/world/chunk/decode.go

```diff
@@ -3,20 +3,26 @@ package chunk
 import (
 	"bytes"
 	"fmt"
+
 	"github.com/df-mc/dragonfly/server/block/cube"
 )
 
 // StateToRuntimeID must hold a function to convert a name and its state properties to a runtime ID.
 var StateToRuntimeID func(name string, properties map[string]any) (runtimeID uint32, found bool)
 
-// NetworkDecode decodes the network serialised data passed into a Chunk if successful. If not, the chunk
```

Comment:
I'd bring back the same docs for this function, so it's shown in IDE's and on the go docs page. It's a short comment anyways so no harm having it twice. Maybe add that it creates a new buffer and calls NetworkDecodeBuffer or something like that